src/avx512bw/avx512bw-constants.inl.cpp

```diff
@@ -0,0 +1,20 @@
+// file included directly
```

Comment:
It is fine but a bit debatable whether it is a great design to have all these global constants when many of them are only used in one specific function.